Jenkinsfile

```diff
@@ -105,7 +105,7 @@ void executeStages(String label) {
         }
 
         stage('Check & Test ' + label) {
-            sh './bin/phpcs'
+            sh './bin/phpcs -s'
```

Comment:
If you want to have a nice summary on the bottom you can also add other reports, for example:
`./bin/phpcs --report=full --report=source --report=summary -s`
Just a tip: https://pear.php.net/manual/en/package.php.php-codesniffer.reporting.php